src/consent-manager-builder/preferences.js

```diff
@@ -33,7 +35,8 @@ export function savePreferences({
   const value = {
     version: 1,
     destinations: destinationPreferences,
-    custom: customPreferences
+    custom: customPreferences,
+    isBannerVisible
```

Comment:
Is this correct? Shouldn't it be `something: isBannerVisible`?

Reply:
Yes, I want to set a property:
```
{isBannerVisible: isBannerVisible}
```

So I just use shortcut for setting key same to variable name.
```
{isBannerVisible}
```